src/story-utils/decorators.js

```diff
@@ -29,3 +30,19 @@ export const RouterDecorator = (Story, {args: {routerArgs = {}}}) => {
     </MemoryRouter>
   );
 };
+
+export const FormikDecorator = (Story, context) => {
+  const {initialValues} = context.parameters;
```

Comment:
I think it's better to namespace this instead of polluting the entire parameters namespace:

```suggestion
  const initialValues = context.parameters?.formik?.initialValues || {};
```